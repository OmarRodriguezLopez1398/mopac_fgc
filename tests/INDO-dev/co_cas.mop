INDO C.A.S.=(3,1)
Carbon Monoxide
INDO 
C  0.000000 0   0.000000 0   0.000000 0
O  1.173000 0   0.000000 0   0.000000 0
