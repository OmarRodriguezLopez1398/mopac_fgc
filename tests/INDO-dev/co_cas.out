 *******************************************************************************
 **                                                                           **
 **                              MOPAC v22.0.6                                **
 **                                                                           **
 *******************************************************************************
 **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **
 **    Visit the DOI location for information on how to cite this program.    **
 *******************************************************************************

                              INDO CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Jul 12 16:25:32 2023  *
 *  INDO       - The INDO Hamiltonian to be used
 * C.A.S.=(N,M)-  1 DOUBLY FILLED LEVELS USED IN A
 *             CAS REF DET GENERATION INVOLVING  3 M.O.'S
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
INDO C.A.S.=(3,1)
 Carbon Monoxide
 INDO
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C          0.00000000      0.00000000      0.00000000   
     2       O          1.17300000      0.00000000      0.00000000   



          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          0.0000      0.0000      0.0000
     2         O          1.1730      0.0000      0.0000

           Empirical Formula: C O  =     2 atoms



      MOLECULAR POINT GROUP   :   C*v 


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 5

 WARNING: INDO only performs single-point calculations

 Summary of symmetry occupancy in each shell:

 SHELL     
    1    5
    v    3
 TOTAL   8


                     MOLECULAR ORBITALS 






    ROOT NO.    1           2           3           4           5           6

           -43.05652   -19.27644   -14.77940   -14.77940   -14.49534     0.09603
  
  
  S  C   1   0.44267    -0.58677    -0.00000     0.00000    -0.60295     0.00000
 PX  C   1   0.36283     0.01825     0.00000     0.00000     0.61061    -0.00000
 PY  C   1  -0.00000    -0.00000     0.48812    -0.00000    -0.00000     0.87278
 PZ  C   1   0.00000    -0.00000    -0.00000     0.48812     0.00000     0.00000
  
  S  O   2   0.77781     0.52976    -0.00000     0.00000    -0.10914     0.00000
 PX  O   2  -0.25962     0.61215    -0.00000     0.00000    -0.50170     0.00000
 PY  O   2   0.00000     0.00000     0.87278     0.00000    -0.00000    -0.48812
 PZ  O   2  -0.00000     0.00000    -0.00000     0.87278     0.00000    -0.00000




    ROOT NO.    7           8

             0.09603    10.49950
  
  
  S  C   1   0.00000     0.31016
 PX  C   1   0.00000     0.70369
 PY  C   1  -0.00000     0.00000
 PZ  C   1   0.87278    -0.00000
  
  S  O   2   0.00000    -0.32009
 PX  O   2   0.00000     0.55333
 PY  O   2  -0.00000    -0.00000
 PZ  O   2  -0.48812    -0.00000


 Reference determinate nber   1 is ALPHA 1 -   5
 CAS excitations amongst orbitals:   5   6   7
 SINGLE excitations FROM orbs   5 to   5 INTO orbs   6 to   7


 CI excitations=    6:        =  6
 The lowest    6 spin-adapted configurations of multiplicity=  1
     sym   eV   cm**-1 -dets- dipole oscilator X FRAG ....Excitations named from first reference determinate
                       tot  #  Debye  strength

   1     -0.000      0.  1  1  0.4656 0.000000 0  0 (        )->(           )
   2      6.886  55542.  2  1  1.8085 0.063444 1  1 (   5    )->(   7       )
   3      6.886  55542.  2  1  1.8085 0.063444 1  1 (   5    )->(   6       )
   4     13.933 112374.  2  1  3.1514 0.000000 2  1 (   5   5)->(   6   7   )
   5     14.294 115286.  1  1  3.1514 0.000000 2  0 (   5   5)->(   6   6   )
   6     14.294 115286.  1  1  3.1514 0.000000 2  0 (   5   5)->(   7   7   )

 Depression of ground-state after CI=       -792. cm**-1 Energy=  -462.4496021 eV
 Dipole moment= -0.483401  0.000000 -0.000000 tot=  0.483401 Debyes


  CI trans.  energy frequency wavelength oscillator--------- polarization---------  dipole  ------ components-----
 st.  symm.    eV      cm - 1       nm      strength       x          y         z       moment     x       y       z 

   2      6.9845001    56334.    177.51  0.056759   0.000000  -0.937754  -0.347301  1.808525  -1.809   0.000  -0.000
   3      6.9845001    56334.    177.51  0.056759  -0.000000   0.347301  -0.937754  1.808525  -1.809   0.000   0.000
   4     14.0308249   113167.     88.37  0.000000                                   3.151416  -3.151   0.000  -0.000
   5     14.0308249   113167.     88.37  0.000000                                   3.151416  -3.151   0.000   0.000
   6     14.8511617   119783.     83.48  0.002670  -1.000000  -0.000000   0.000000  3.133649  -3.134   0.000  -0.000

 Polarizability (au  ) xx=    0.03 xy=   -0.00 yy=    2.58 xz=    0.00 yz=    0.00 zz=    2.58
 Polarizability (A**3) xx=    0.00 xy=   -0.00 yy=    0.38 xz=    0.00 yz=    0.00 zz=    0.38


 Major CI contributions to CI states

State    1  0.0000     CI coeff  CI percent
    Config       1  -0.99668691  0.99338481
    Config       5   0.05751171  0.00330760
    Config       6   0.05751171  0.00330760
 Total coeff printed             1.00000000
  
State    2  6.9845     CI coeff  CI percent
    Config       2  -0.34730101  0.12061799
    Config       3   0.93775370  0.87938201
 Total coeff printed             1.00000000
  
State    3  6.9845     CI coeff  CI percent
    Config       2  -0.93775370  0.87938201
    Config       3  -0.34730101  0.12061799
 Total coeff printed             1.00000000
  
State    4 14.0308     CI coeff  CI percent
    Config       5  -0.70710678  0.50000000
    Config       6   0.70710678  0.50000000
 Total coeff printed             1.00000000
  
State    5 14.0308     CI coeff  CI percent
    Config       4   1.00000000  1.00000000
 Total coeff printed             1.00000000
  
State    6 14.8512     CI coeff  CI percent
    Config       1   0.08133384  0.00661519
    Config       5   0.70476408  0.49669240
    Config       6   0.70476408  0.49669240
 Total coeff printed             1.00000000
  

 -------------------------------------------------------------------------------
 INDO C.A.S.=(3,1)
 Carbon Monoxide
 INDO


     NO PARAMETERS MARKED FOR OPTIMIZATION, SO 1SCF WAS USED  
     SCF FIELD WAS ACHIEVED                                   


                              INDO CALCULATION
                                                       MOPAC v22.0.6 MacOS
                                                       Wed Jul 12 16:25:32 2023


          COSMO AREA              =         57.92 SQUARE ANGSTROMS
          COSMO VOLUME            =         40.40 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         14.495342 EV
          HOMO LUMO ENERGIES (EV) =        -14.495  0.096
          NO. OF FILLED LEVELS    =          5
          MOLECULAR WEIGHT        =         28.0104         POINT GROUP:  C*v 

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            O     2    C     1     1.17300
            O     2    C     1     0.00000
            C     1    O     2     0.00000


          SCF CALCULATIONS        =          1

          WALL-CLOCK TIME         =      0.043 SECONDS
          COMPUTATION TIME        =      0.039 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C          0.00000000      0.00000000      0.00000000   
     2       O          1.17300000      0.00000000      0.00000000   

                             CARTESIAN COORDINATES

   1    C        0.000000000     0.000000000     0.000000000
   2    O        1.173000000     0.000000000     0.000000000


           Empirical Formula: C O  =     2 atoms



      MOLECULAR POINT GROUP   :   C*v 


                  EIGENVALUES  
 -43.05652 -19.27644 -14.77940 -14.77940 -14.49534   0.09603   0.09603  10.49950


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.228558        3.7714     1.80762     1.96383
    2          O          -0.228558        6.2286     1.79506     4.43350
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -1.288     0.000     0.000     1.288
 HYBRID         0.828    -0.000    -0.000     0.828
 SUM           -0.460    -0.000    -0.000     0.460


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.80762   1.00993   0.47695   0.47695
    2  O   1.79506   1.38739   1.52305   1.52305

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.07 SECONDS

 == MOPAC DONE ==
